official-allow_functionality.json

```diff
@@ -229,8 +229,43 @@
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.s3-us-west-2.amazonaws.com"}},
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.cloudflare.com"}},
 		{"o":{"h":"*.hansje.net"},"d":{"h":"*.cargocollective.com"}},
-		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}}
+		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}},
+		{"o":{"h":"*.eurobuch.com"},"d":{"h":"*.lehmanns.de"}},
+		{"o":{"h":"*.emp.de"},"d":{"h":"*.acfrg.com"}},
+		{"o":{"h":"*.microsoft.com"},"d":{"h":"*.s-msft.com"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.trackersimulator.org"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.eviltracker.net"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.do-not-tracker.org"}},
+		{"o":{"h":"*.shortfil.ms"},"d":{"h":"*.eay.cc"}},
```

Comment:
> another solution (it was suggested somewhere) would be separate subscriptions that users would be able to enable separately (eg Multimedia content subscription would whitelist vimeo.com vimeocdn.com youtube.com ytimg.com dailymotion.com dmcdn.com soundcloud.com sndcdn.com...)

You've suggested it in #21 and https://github.com/RequestPolicyContinued/requestpolicy/issues/338. There you've talked about "bundles". However, it would actually be a "subscription list", no?

I'm somewhat confused by the word "bundle". The current most important issue should be to un-hardcode the available subscriptions (see [rp#678](https://github.com/RequestPolicyContinued/requestpolicy/issues/678)). Then we won't work with bundles, but instead with "subscription lists", just like we do now.

I understand, and agree, that those "bundles" you've suggested have big potential to be a very interesting feature. However, other things have to come first. So why not use the `allow_embedded` subscription list for the "Media/Multimedia" content for now?